pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 79
```

Comment:
When using `black` we need to have a `pyproject.toml` to specify our default 80 char line length